frappe_graphql/api.py

```diff
@@ -0,0 +1,113 @@
+from graphql import GraphQLError
+
+import frappe
+from .graphql import execute
+
+
+@frappe.whitelist(allow_guest=True)
+def execute_gql_query():
+    query, variables, operation_name = get_query()
+    output = execute(
+        query=query,
+        variables=variables,
+        operation_name=operation_name
+    )
+
+    frappe.local.response = output
+    if len(output.get("errors", [])):
+        frappe.db.rollback()
+        log_error(query, variables, operation_name, output)
+        frappe.local.response["http_status_code"] = 400
```

Comment:
extract from exception thrown is better right ? @assemmarwan ? frappe.throw default is 
![image](https://user-images.githubusercontent.com/41537985/115149908-3a06ab00-a077-11eb-821b-ee6988689665.png)

Reply:
ok. i will fetch the max that was thrown

Reply:
@assemmarwan @e-lobo updated